k8s-deep-dive-charly-scratchpad/06-k8s-dashboard.md

```diff
@@ -16,9 +16,9 @@ You can also monitor the different components of the Control Plane:
 * [Kubernetes Controller Manager](https://app.datadoghq.com/screen/integration/30271/kubernetes-controller-manager)
 * [ETCD](https://app.datadoghq.com/screen/integration/30289/etcd-overview)
 
-Most of these dashboards display data from the OpenMetrics endpoints of the applications (apiserver, kubelet, etcd ...) but the agent is also using [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics). Also known as KSM, `kube-state-metrics` is a service that watches the Kubernetes API and generates metrics for the state of objects. You can find the official Datadog documentation [here](https://docs.datadoghq.com/integrations/kubernetes/#setup-kubernetes-state) for the check.
+Most of these dashboards display data from metrics available on `/metrics` endpoints of the HTTP server running in these applications (e.g. apiserver, kubelet, etcd). In addition, Datadog Agent uses [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics), also known as KSM. `kube-state-metrics` is a service that watches the Kubernetes API and generates metrics for the state of objects. You can find the official Datadog documentation [here](https://docs.datadoghq.com/integrations/kubernetes/#setup-kubernetes-state) for the check.
```

Comment:
why don't we mention the openmetrics endpoints ?

Reply:
we can mention it when presenting but otherwise I think that it is confusing. I'll add "Prometheus /metrics endpoint" instead if that's ok. I can explain more about the context here